src/sprite.rs

```diff
@@ -228,8 +228,18 @@ impl Spritesheet {
 }
 
 /// Returns the name (unique id within a spritesheet) taken from a file.
-pub fn sprite_name(path: &Path) -> String {
-    format!("{}", path.file_stem().unwrap().to_string_lossy())
+pub fn sprite_name(path: &Path, base_path: &Path) -> String {
```

Comment:
Now it's doing something a little more complicated, I think this function's a perfect candidate for a unit test or two (I'm holding you to a higher standard than I hold myself, sorry!) Do you fancy adding some? One for when base_path has no impact, one when it does

Reply:
Addressed in 73fda65003dc95ce4a639e30c65a1ad57d4d6aa4